lib/topological_inventory/scheduler/database_tools.rb

```diff
@@ -0,0 +1,24 @@
+require 'topological_inventory/scheduler/logging'
+require 'active_support/core_ext/integer/time'
+
+module TopologicalInventory
+  module Scheduler
+    class DatabaseTools
+      include Logging
+
+      REFRESH_STATES_TIME_THRESHOLD = 1.week.freeze
+
+      def remove_old_records
```

Comment:
This should probably be a class method rather than an instance method, no state is stored.

Reply:
@lindgrenj6 I recommended the opposite because of the similar style with other classes. With class method you can't `include Logging` like in all other services we have